### PR TITLE
[NUI.Wearable] Remove build warning

### PR DESCRIPTION
**Pull request by hinohie**
Remove dotnet build warnings for Tizen.NUI.Wearable.

- Add comment for public API
- Remove some unused private value/method

### CODE REVIEW

File: src/Tizen.NUI.Wearable/src/public/CircularPagination.cs [hunk 1]
```diff
@@ -48,7 +48,7 @@ public class CircularPagination : Control
         private bool isOddNumber = true;
         private bool uninitializedLeftIndicator = true; // Need it when the indicators are asymmetry and the right indicator count is set earlier than left one.
         private Animation selectAnimation = null;
-        private bool isNeedAnimation = false; // TODO : Animation will support using override function later.
+        //private bool isNeedAnimation = false; // TODO : Animation will support using override function later.
```
Comment (Seoyeon2Kim) [inline src/Tizen.NUI.Wearable/src/public/CircularPagination.cs:51]: It's not used anywhere, then can we remove it at all?

**Review by Seoyeon2Kim — COMMENTED** [review]
Thank you for fixing the warning, but could you update the description of this PR?
(even though Wearable repo is not used anymore)

File: src/Tizen.NUI.Wearable/src/public/CircularPagination.cs (hunk 1 above)
Comment (hinohie) [reply src/Tizen.NUI.Wearable/src/public/CircularPagination.cs:51]: I'm not sure what TODO comment writer want to do in future. So I just want to keep this comment. That's why I dont remove this line. What do you think about it? Could I remove this line?

Comment (hinohie) [reply src/Tizen.NUI.Wearable/src/public/CircularPagination.cs:51]: There is also an unused API in this file. If we can remove unused values, Ill also try to clean up soon

Comment (Seoyeon2Kim) [reply src/Tizen.NUI.Wearable/src/public/CircularPagination.cs:51]: From what I've remembered, Circular Pagination had a requirement to select whether animation is working or not.
That's why this variable was added.
However, `NUI.Wearable` is meaningless now :/ and we can add the unused variables later if this class will be used someday.

So, Please remove the unused values in this file

**Review by Seoyeon2Kim — APPROVED** [review]
LGTM!!!!!!!!!
Thank you for updating the files.

**Pull request merged**